drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -227,12 +227,21 @@ function DragAndDropTemplates(configuration) {
     };
 
     var feedbackTemplate = function(ctx) {
-        var feedback_display = ctx.feedback_html ? 'block' : 'none';
         var properties = { attributes: { 'aria-live': 'polite' } };
+        var messages = ctx.overall_feedback_messages || [];
+        var feedback_display = messages.length > 0 ? 'block' : 'none';
+        var feedback_messages = messages.map(function(message) {
+            var selector = "p.message";
+            if (message.message_class) {
+                selector += "."+message.message_class;
+            }
+            return h(selector, {innerHTML: message.message}, []);
```

Comment:
@cahrens The basic idea is that author can use HTML markup in feedback messages, and it should not be escaped (yes, I've missed "not" in previous comment). It does not need to work exactly like this, but this is how it worked previously - I haven't changed the approach, only the code to implement it.